ember-toucan-core/src/components/form/checkbox-field.hbs

```diff
@@ -0,0 +1,41 @@
+<div
+  class="flex flex-col"
+  data-root-field={{if @rootTestSelector @rootTestSelector}}
+>
+  <Form::Field as |field|>
+    <field.Control
+      class="rounded-sm p-1 {{if @error 'shadow-error-outline'}}"
+      data-control
+    >
+
+      <label class="flex flex-row space-x-3" for={{field.id}}>
```

Comment:
Can't use field's yielded label as we need different leading as well as use a different text color for checkbox labels. I verified with Mary and David this is the expected look and that we don't want to use the styles from field.Label